cmd/theatre-envconsul/main.go

```diff
@@ -43,8 +43,8 @@ var (
 	exec             = app.Command("exec", "Authenticate with vault and exec envconsul")
 	execVaultOptions = newVaultOptions(exec)
 	execConfigFile   = exec.Flag("config-file", "app config file").String()
-	execCommand      = exec.Flag("command", "Command to execute").Required().String()
 	execInstallPath  = exec.Flag("install-path", "Path containing installed binaries").Default(defaultInstallPath).String()
+	execCommand      = exec.Arg("command", "Command to execute").Required().String()
```

Comment:
I think you need to make this `Strings()` right? So that it's an array?

If you do that, compiler errors should help you figure out what remains to be changed